### PR TITLE
[mono][debugger] Fix step on ios when using interpreter

### PR DESCRIPTION
**Pull request by thaystg**
On `save_seq_points` function we loop in this `td->basic_blocks` list, but this block was not being added so we could not step correctly on iOS.

### CODE REVIEW

File: src/mono/mono/mini/interp/transform.c [hunk 1]
```diff
@@ -3576,6 +3576,12 @@ interp_field_from_token (MonoMethod *method, guint32 token, MonoClass **klass, M
 	return field;
 }
 
+static void 
+add_bb_to_basic_blocks(TransformData *td, InterpBasicBlock *bb)
```
Comment (BrzVlad) [inline src/mono/mono/mini/interp/transform.c:3580]: I don't really see the usefulness of this method

**Review by BrzVlad — APPROVED** [review]

File: src/mono/mono/mini/interp/transform.c (hunk 1 above)
Comment (thaystg) [reply src/mono/mono/mini/interp/transform.c]: I removed the function.****

**Comment by marek-safar** [comment]
Should this be backported?

**Comment by thaystg** [comment]
> Should this be backported?
I think so. I'm not sure if to 6.0 or to 6.0maui.

**Comment by marek-safar** [comment]
6.0maui at this point

**Pull request merged**